app/src/main/java/com/blogspot/e_kanivets/moneytracker/entity/RecordItem.kt

```diff
@@ -4,7 +4,7 @@ sealed class RecordItem {
 
     data class Header(val date: String) : RecordItem()
 
-    data class Record(val title: String, val categoryName: String, val fullPrice: Double, val currency: String, val isIncome: Boolean) : RecordItem() {
-        constructor(record: com.blogspot.e_kanivets.moneytracker.entity.data.Record) : this(record.title, record.category?.name?.toString().orEmpty(), record.fullPrice, record.currency, record.isIncome)
+    data class Record(val title: String, val categoryName: String, val notes: String, val fullPrice: Double, val currency: String, val isIncome: Boolean) : RecordItem() {
+        constructor(record: com.blogspot.e_kanivets.moneytracker.entity.data.Record) : this(record.title, record.category?.name?.toString().orEmpty(), record.notes, record.fullPrice, record.currency, record.isIncome)
```

Comment:
If you have existing records (before the database migration), they don't have notes (`null`), and the app crashes on the launch here. To fix the behavior you need to provide a default value (empty string?). To test the fix, you can add a few records on the current Google Play version, then install the one from this branch.

Reply:
Fixed with [610cc48](https://github.com/xorum-io/open_money_tracker/pull/34/commits/610cc4834b30bc81351585fc0b4588bca33a69a2)